api/live.js

```diff
@@ -26,6 +26,7 @@ const actions = {
 
 router.post("/dispatch", async (req, res) => {
   const { userId, actionId } = { ...req.body };
+  console.log({ userId, actionId });
```

Comment:
What's our stance on `console.log`?

Reply:
oh yeet will remove -- still considering this WIP but will make sure to remove that